dataservice/api/family_relationship/models.py

```diff
@@ -35,62 +35,101 @@ class FamilyRelationship(db.Model, Base):
     """
     __tablename__ = 'family_relationship'
     __prefix__ = 'FR'
-    __table_args__ = (db.UniqueConstraint('participant_id', 'relative_id',
-                                          'participant_to_relative_relation',
-                                          'relative_to_participant_relation'),)
+    __table_args__ = (db.UniqueConstraint(
+        'participant1_id', 'participant2_id',
+        'participant1_to_participant2_relation',
+        'participant2_to_participant1_relation'),)
     external_id = db.Column(db.Text(),
                             doc='external id used by contributor')
-    participant_id = db.Column(
+    participant1_id = db.Column(
         KfId(),
         db.ForeignKey('participant.kf_id'),
         nullable=False,
         doc='kf_id of one participant in the relationship')
 
-    relative_id = db.Column(
+    participant2_id = db.Column(
         KfId(),
         db.ForeignKey('participant.kf_id'),
         nullable=False,
         doc='kf_id of the other participant in the relationship')
 
-    participant_to_relative_relation = db.Column(db.Text(), nullable=False)
+    participant1_to_participant2_relation = db.Column(db.Text(),
+                                                      nullable=False)
 
-    relative_to_participant_relation = db.Column(db.Text())
+    participant2_to_participant1_relation = db.Column(db.Text())
```

Comment:
Is this nullable because it's assumed to be populated by the other relationship? Maybe it should still be explicitly non-nullable anyway.

Reply:
Think I'm going to leave this out for now since the migration could break things. First we can get this PR deployed as is and then check the db in all envs to ensure that there are no nulls in `participant2_to_participant1_relation`, if there are any, populate w a placeholder. Then we can deploy a change to make this column non-nullable